### PR TITLE
Feat/home page

### PR DESCRIPTION
**Pull request by BalajiHarinath**
Added Home page and video listing page along with the category filter functionality. Please review.

Files to be reviewed:
[components/CardLoader/CardLoader.jsx](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/CardLoader/CardLoader.jsx)
[components/CardLoader/CardLoader.css](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/CardLoader/CardLoader.css)
[components/CardVideo/CardVideo.jsx](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/CardVideo/CardVideo.jsx)
[components/CardVideo/CardVideo.css](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/CardVideo/CardVideo.css)
[components/HeaderNav/HeaderNav.jsx](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/HeaderNav/HeaderNav.jsx)
[components/HeaderNav/HeaderNav.css](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/HeaderNav/HeaderNav.css)
[components/SidebarNav/SidebarNav.jsx](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/SidebarNav/SidebarNav.jsx)
[components/SidebarNav/SidebarNav.css](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/components/SidebarNav/SidebarNav.css)
[pages/Home/Home.jsx](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/pages/Home/Home.jsx)
[pages/Home/Home.css](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/pages/Home/Home.css)
[pages/Videos/Videos.jsx](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/pages/Videos/Videos.jsx)
[pages/Videos/Videos.css](https://github.com/BalajiHarinath/Video-library-react/blob/feat/home-page/src/pages/Videos/Videos.css)

You can ignore all other files.

[Live Link](https://easy-tv-feat-home.netlify.app/)

### CODE REVIEW

File: src/components/CardVideo/CardVideo.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+              {channel}
+            </div>
+            <span className="text-xs">{views} views</span>
+          </div>
+        </div>
+        <button
+          className={`${
+            isMoreOptions ? "btn-more-vert-card-active" : " "
+          } btn-more-vert-card btn-transparent flex flex-align-center flex-justify-center`}
+          onClick={() => setIsMoreOptions(!isMoreOptions)}
```
Comment (mikesch803) [inline src/components/CardVideo/CardVideo.jsx:72]: improvement : just close the previous card option when user click on another card option

File: src/components/SidebarNav/SidebarNav.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import "./SidebarNav.css";
+import { Link, useLocation } from "react-router-dom";
+import { SidebarNavData } from "./SidebarNavData";
+
+export const SidebarNav = () => {
+  const location = useLocation();
+  return (
+    <aside className="container-sidebar">
+      <ul className="list-style-none">
+        {SidebarNavData.map(({ _id, name, icon, link }) => {
```
Comment (mikesch803) [inline src/components/SidebarNav/SidebarNav.jsx:10]: appreciation: not repeating the each aside option

File: src/components/SidebarNav/SidebarNav.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          return (
+            <Link
```
Comment (mikesch803) [inline src/components/SidebarNav/SidebarNav.jsx]: suggestion : you can use the navlink, as it has isActive attribute

**Review by mikesch803 — APPROVED** [review]
i liked that you have used the loaders for cards, and chips, overall code is also good, just few suggestions

File: src/components/SidebarNav/SidebarNav.jsx (hunk 3 above)
Comment (BalajiHarinath) [reply src/components/SidebarNav/SidebarNav.jsx]: Done, used NavLink instead of Link

File: src/components/CardVideo/CardVideo.jsx (hunk 1 above)
Comment (BalajiHarinath) [reply src/components/CardVideo/CardVideo.jsx:72]: Done. Thanks for pointing this.

**Review by b0llu — APPROVED** [review]
Clean code. Was able to understand each line. Really well written. nothing to suggest. You can merge this PR

**Pull request merged**